engine/src/main/java/org/terasology/rendering/world/RenderableWorldImpl.java

```diff
@@ -75,14 +78,19 @@
     private int statVisibleChunks;
     private int statIgnoredPhases;
 
-    public RenderableWorldImpl(WorldProvider worldProvider, ChunkProvider chunkProvider, GLBufferPool bufferPool, Camera playerCamera, Camera shadowMapCamera) {
+    public RenderableWorldImpl(WorldProvider worldProvider,
+                               ChunkProvider chunkProvider,
+                               GLBufferPool bufferPool,
+                               Camera playerCamera,
+                               DirectedAcyclicClassGraph<RenderNode> dag) {
+
         this.worldProvider = worldProvider;
         this.chunkProvider = chunkProvider;
         chunkTessellator = new ChunkTessellator(bufferPool);
         chunkMeshUpdateManager = new ChunkMeshUpdateManager(chunkTessellator, worldProvider);
 
         this.playerCamera = playerCamera;
-        this.shadowMapCamera = shadowMapCamera;
+        this.dag = dag;
```

Comment:
Let's use the class and instance name to communicate as much information as possible. Let's call it RenderingPipeline.